site_root/input/ImplementationGuide-NCPI-Project-Forge.json

```diff
@@ -141,6 +141,62 @@
         "reference": {
           "reference": "StructureDefinition/age-at-event"
         }
+      },
+      {
+        "description": "NCPI Project Forge ValueSet phenotype-codes",
+        "exampleBoolean": false,
+        "name": "NCPI Project Forge ValueSet/phenotype-codes",
+        "reference": {
+          "reference": "ValueSet/phenotype-codes"
+        }
+      },
+      {
+        "description": "NCPI Project Forge Observation hpo-1, Profiles: ncpi-phenotype",
+        "exampleBoolean": true,
+        "name": "NCPI Project Forge Observation/hpo-1",
+        "reference": {
+          "reference": "Observation/hpo-1"
+        }
+      },
```

Comment:
How about assigning `ph-001` as a resource ID since this example is an instantiation of a phenotype observation rather than that of HPO?